register.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 func init() {
-	output.RegisterExtension("output-prometheus-remote", func(p output.Params) (output.Output, error) {
+	output.RegisterExtension("xk6-prometheus-rw", func(p output.Params) (output.Output, error) {
```

Comment:
We don't want to use `experimental-*` here?

And does `xk6-*` make sense, considering this extension will be imported directly by k6, without using xk6?

Reply:
We can't use `experimental-`, since that will already be built into k6 and cause an error. 

In other extensions, we kept the xk6 module registration even after we bundled them into the core as `experimental` because some users probably already had a setup that used them as xk6 extensions. In this case, since we are changing the extension name and all of its parameters, it probably males less sense, so I am fine either way - leave it as it was, change it, doesn't really matter all that much :sweat_smile: 

Though some xk6 registration _should_ remain, since this is still going to be an xk6 extension in its own repo, even after it is bundled in `experimental`. We will make changes here and only occasionally merge them back in k6, so some user might want to use the latest and greatest version of the extension through xk6.